src/OrchardCore/OrchardCore.ContentManagement.Abstractions/ContentItemExtensions.cs

```diff
@@ -95,6 +95,16 @@ public static ContentItem Merge(this ContentItem contentItem, object properties,
                 contentItem.DisplayText = props[nameof(contentItem.DisplayText)].ToString();
             }
 
+            if (props.ContainsKey(nameof(contentItem.Owner))) {
+                contentItem.Owner = props[nameof(contentItem.Owner)].ToString();
+                contentItem.Author = props[nameof(contentItem.Owner)].ToString();
```

Comment:
Why assign the owner to the author?

Reply:
If there is no Author provided then the Author is the Owner. Then if you look afterward, if it finds a provided Author it will replace the previously assigned Author = Owner.

Reply:
![Sans titre](https://user-images.githubusercontent.com/3228637/73891412-2ee4a800-4842-11ea-9c9d-13d54bbedf38.png)

Here you can see some content items that we're created from the admin UI and the Author is assigned to the same user than the Owner.

Reply:
A cloned content item is not assigning an Author which is an issue.

![image](https://user-images.githubusercontent.com/3228637/73891492-58053880-4842-11ea-86ec-4e6b9f557831.png)

Reply:
Here we can also see that a Menu created from a recipe has no Owner/Author

![Sans titre](https://user-images.githubusercontent.com/3228637/73891657-b205fe00-4842-11ea-9459-b77d6a520645.png)